projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/date-time-field/date-time-field.component.html

```diff
@@ -9,6 +9,10 @@
       </span>
     </button>
     <ngx-mat-datetime-picker panelClass="lf-datetime-picker-overlay" [showSeconds]=true [enableMeridian]="shouldEnableMeridian" #picker>
+      <ngx-mat-datepicker-actions>
+        <button mat-button ngxMatDatepickerCancel>Cancel</button>
```

Comment:
Are these still just plain buttons? I'm okay with it just being an "Apply" button instead of checkbox, but it should be styled